tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -194,153 +219,158 @@ async def test_semi_manual_upgrade(self, bft_network):
         await self.validate_state_consistency(skvbc, key, val)
         await skvbc.write_known_kv()
 
-    @with_trio
```

Comment:
Hi @yontyon , please use `@unittest.skip()` to disable tests, instead of commenting them out. Thanks!

Reply:
These tests are not relevant anymore. I will remove them completely.